extensions/apps/extensions/src/components/pages/extension-creation-page.tsx

```diff
@@ -1,29 +1,38 @@
-import React, { useState } from 'react';
+import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { useNavigate } from '@tanstack/react-router';
-import routes, { MY_EXTENSIONS } from '../../routes';
-import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
-import { useCreateAppMutation } from '@akashaorg/ui-awf-hooks/lib/generated/apollo';
-import getSDK from '@akashaorg/awf-sdk';
+import routes, { CREATE_EXTENSION, MY_EXTENSIONS } from '../../routes';
+import { useRootComponentProps, useAkashaStore } from '@akashaorg/ui-awf-hooks';
 import { NotificationEvents, NotificationTypes } from '@akashaorg/typings/lib/ui';
 import Card from '@akashaorg/design-system-core/lib/components/Card';
 import Stack from '@akashaorg/design-system-core/lib/components/Stack';
 import Text from '@akashaorg/design-system-core/lib/components/Text';
 import Divider from '@akashaorg/design-system-core/lib/components/Divider';
-import AppCreationForm, {
-  FieldName,
-} from '@akashaorg/design-system-components/lib/components/AppCreationForm';
+import AppCreationForm from '@akashaorg/design-system-components/lib/components/AppCreationForm';
+import { NotConnnected } from '../not-connected';
```

Comment:
You can use ErrorLoader component from the ds-core

Reply:
yes, I will change it, the error loader was updated in Jose's PR